src/components/results/common/global-filter/global-filter-paper.tsx

```diff
@@ -0,0 +1,201 @@
+/**
+ * Copyright (c) 2025, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import { Box, Button, Chip, Grid, ListItem, ListItemButton, Paper, Typography } from '@mui/material';
+import {
+    getResultsGlobalFiltersChipStyle,
+    GLOBAL_FILTERS_CELL_HEIGHT,
+    IMPORT_FILTER_HEIGHT,
+    resultsGlobalFilterStyles,
+} from './global-filter-styles';
+import { FormattedMessage, useIntl } from 'react-intl';
+import FileUploadIcon from '@mui/icons-material/FileUpload';
+import { PropsWithChildren, useCallback, useContext, useMemo } from 'react';
+import ListItemText from '@mui/material/ListItemText';
+import List from '@mui/material/List';
+import { FilterType } from '../utils';
+import { GlobalFilter } from './global-filter-types';
+import { getOptionLabel, RECENT_FILTER } from './global-filter-utils';
+import { useLocalizedCountries } from '../../../utils/localized-countries-hook';
+import ClickAwayListener from '@mui/material/ClickAwayListener';
+import {
+    DirectoryItemSelector,
+    ElementAttributes,
+    ElementType,
+    fetchElementsInfos,
+    mergeSx,
+    TreeViewFinderNodeProps,
+} from '@gridsuite/commons-ui';
+import { GlobalFilterContext } from './global-filter-context';
+import { UUID } from 'crypto';
+
+const XS_COLUMN1: number = 3.5;
+const XS_COLUMN2: number = 4;
+const XS_COLUMN3: number = 4.5;
+
+function GlobalFilterPaper({ children }: Readonly<PropsWithChildren>) {
+    const {
+        setOpenedDropdown,
+        directoryItemSelectorOpen,
+        setDirectoryItemSelectorOpen,
+        filterGroupSelected,
+        setFilterGroupSelected,
+        selectedGlobalFilters,
+        setSelectedGlobalFilters,
+        onChange,
+    } = useContext(GlobalFilterContext);
+    const { translate } = useLocalizedCountries();
+    const intl = useIntl();
+
+    const categories: string[] = useMemo(() => [RECENT_FILTER, ...Object.values(FilterType)], []);
+
+    const filtersMsg: string = useMemo(
+        () =>
+            intl.formatMessage(
+                {
+                    id:
+                        selectedGlobalFilters.length < 2
+                            ? 'results.globalFilter.activeFilter'
+                            : 'results.globalFilter.activeFilters',
+                },
+                { filtersCount: selectedGlobalFilters.length }
+            ),
+        [intl, selectedGlobalFilters.length]
+    );
+
+    const addSelectedFilters = useCallback(
+        (values: TreeViewFinderNodeProps[] | undefined) => {
+            if (!values) {
+                return;
+            }
+
+            fetchElementsInfos(values.map((value) => value.id as UUID)).then((elements: ElementAttributes[]) => {
```

Comment:
Not needed : id IS a UUID anyway : 
```suggestion
            fetchElementsInfos(values.map((value) => value.id)).then((elements: ElementAttributes[]) => {
```

Reply:
DONE in https://github.com/gridsuite/gridstudy-app/pull/2791/commits/be94b006d2c6523fcc413269097da148fa102383